src/Client.js

```diff
@@ -96,8 +96,14 @@ class Client extends EventEmitter {
     }
 
     // Attach to server sent events and re broadcast.
-    this._events = new Eventsource([this.URL,
-      SERVER_API.EVENTS_PATH].join('/'));
+    // Include headers as query strings.
+    const queryString = Object.keys(this.headers).map((key) => {
```

Comment:
Maybe it would be safer to put this in an if (this.headers) check?

Reply:
Sure I can add that check.